### ISSUE TITLE
The method 'getElementForInheritedWidgetOfExactType' was called on null

### ISSUE THREAD

**Issue by mehmetf** [flutter/flutter#64272]
Stacktrace:

Thread 0 main thread CRASHED

```
NoSuchMethodError: The method 'getElementForInheritedWidgetOfExactType' was called on null.
Receiver: null

Tried calling: getElementForInheritedWidgetOfExactType()
at Actions._visitActionsAncestors	(actions.dart:457)
at Actions.invoke	(actions.dart:608)
at ShortcutManager.handleKeypress	(shortcuts.dart:330)
at _ShortcutsState._handleOnKey	(shortcuts.dart:472)
at FocusManager._handleRawKeyEvent	(focus_manager.dart:1625)
at RawKeyboard._handleKeyEvent	(raw_keyboard.dart:594)
at BasicMessageChannel.setMessageHandler.<anonymous closure>	(platform_channel.dart:75)
at BasicMessageChannel.setMessageHandler.<anonymous closure>	(platform_channel.dart:74)
at _DefaultBinaryMessenger.handlePlatformMessage	(binding.dart:283)
at _invoke3.<anonymous closure>	(hooks.dart:282)
at _rootRun	(zone.dart:1190)
at _CustomZone.run	(zone.dart:1093)
at _CustomZone.runGuarded	(zone.dart:997)
at _invoke3	(hooks.dart:281)
at _dispatchPlatformMessage	(hooks.dart:156)
```

**Comment by HansMuller** [comment]
CC @gspencergoog

**Comment by gspencergoog** [comment]
Can you give me some more context?  This looks like the actions widget doesn't have any ancestors, which is possible, but unlikely.

I'll put together a PR to prevent this from throwing.

**Comment by gspencergoog** [comment]
Actually, in trying to build a regression test for this, I don't see how this can happen. In order for this to happen, there needs to be an `_ActionsMarker` at the root of the widget tree, which is a private class that is only added by a parent `Actions` class (which means it won't be at the root).

I can protect against this throwing for this reason, but I don't think I can make a regression test for it without more information.

**Comment by mehmetf** [comment]
Could this be caused by some race between the widget tree being disposed and the key being handled? During navigation perhaps?

**Comment by gspencergoog** [comment]
Yes, I suppose so. From what I can tell, if this was a debug build it would have asserted long before it got to this state, probably in `ShortcutManager.handlKeypress`.

**Comment by mehmetf** [comment]
We don't have a repro for this primarily because it is happening in the wild (a release build).

**Comment by HansMuller** [comment]
Although we're going to patch this problem with #64472, it's not really resolved until we find a way to reproduce the problem. In other words #64472 should be viewed as temporary, pending further analysis.

@mehmetf - please help us out by encouraging the devs who ran into this issue to continue trying to find a reproducible test case.

**Comment by singun-lxd** [comment]
I have the same problem when I upgrade to flutter 1.2.4
